Disaster Recovery/SafeHaven 5 General/SafeHaven5.0.1-Release-Notes.md

```diff
@@ -24,6 +23,11 @@ This is a patch release for [SafeHaven 5.0.0](SafeHaven5.0.0-Release-Notes.md) w
 ### Upgrade Path
 **NOTE**: Upgrade from SafeHaven-4 to SafeHaven-5 will be a fresh new installation.
 
+### SafeHaven Most Recent Releases
+Please take a look at the link below to download the latest available release  
+https://www.ctl.io/knowledge-base/disaster-recovery/overview/most-recent-safehaven-release-updates/

```

Comment:
change to [Most recent updates](../../Disaster Recovery/Overview/most-recent-safehaven-release-updates.md)